dali/pipeline/operator/op_spec.h

```diff
@@ -174,6 +174,11 @@ class DLL_PUBLIC OpSpec {
     return this->SetArg<std::string>(name, c_str);
   }
 
+  // Forward to string implementation
+  DLL_PUBLIC inline OpSpec& SetArg(const string &name, const char *c_str) {
```

Comment:
Without this overload, passing `const char *` (but not a literal, which is properly handled above) resulted in a run-time error - there was an attempt to set a pointer value to the argument, which was a bug.